src/cloudflare/internal/pipeline-transform.ts

```diff
@@ -57,26 +58,39 @@ enum Format {
   JSON_STREAM = 'json_stream', // jsonl
 }
 
-export class PipelineTransformImpl extends entrypoints.WorkerEntrypoint {
+type PipelineBatchMetadata = {
+  pipelineId: string;
+  pipelineName: string;
+};
+
+type PipelineRecord<T> = {
+  body: T;
```

Comment:
We can probably drop the generic here for `Record<string,unknown>` (which is a tighter type than `object` since `Array` is also `object`. 

```suggestion
  body: Record<string, unknown>;
```